prepare-rootfs/run.sh

```diff
@@ -63,14 +64,12 @@ Miscellaneous:
                        The list may be filtered with -k
 
   -h, --help           display this help message and exit"
-
+EOF
 	case "$1" in
 		out)
-			echo "$USAGE_STRING"
 			exit 0
 			;;
 		err)
-			echo "$USAGE_STRING" >&2
```

Comment:
Okay, so earlier we were emitting the string to stderr, now we are no longer? If the title of the diff is "suppress warnings" then I infer that semantics are preserved, no? Unsure how to reconcile these two statements, to be honest. Can you please keep diffs focused on a single purpose and *explain* why something is done. I will not be the only one reading this code down the line and posing similar questions, so it would really help if we explained changes being made.

Reply:
Yes, this was a mistake.
The complaint here is that the string was not terminated. Ideally we need to put a `\` at every end of line, which will clutter the file.
I can restore the stderr output with a simple change